go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/UpCloudLtd/terraform-provider-upcloud
 go 1.14
 
 require (
-	github.com/UpCloudLtd/upcloud-go-api v0.0.0-20210127073406-2964ed7e5972
+	github.com/UpCloudLtd/upcloud-go-api v0.0.0-20210525102709-661419a1956b
```

Comment:
why this is not the actual tag?

Reply:
This is how it ended up with `go get github.com/UpCloudLtd/upcloud-go-api@4.0.0` - I'm guessing it's related to the `v0.0.0` prefix and my working theory based on that is that `go` (the tool) expects git tags to be `vX.Y.Z` rather than `X.Y.Z` but did not delve deeper into it for the time being.

Reply:
AFAIK the major version should be included as a suffix for the import path. So `github.com/UpCloudLtd/upcloud-go-api/v4`. Ref: https://golang.org/ref/mod